pytrip/utils/spc2pdf.py

```diff
@@ -168,19 +173,17 @@ def main(args=sys.argv[1:]):
                 fig, ax = plt.subplots()
                 ax.set_title("Spectrum @ {:3.3f} cm".format(depth_step))
                 ax.set_xlabel("Energy [MeV/amu]")
-                ax.set_ylabel("Fluence [a.u.]")
```

Comment:
That is not fluence.

Rather "Number of particles [/MeV]".

Reply:
Number of particles [ / MeV / primary ]

Reply:
Here it is not longer /MeV as I multiply by energy bin widths to obtain total number of particles crossing certain depth step. But I agree that it is not fluence

Reply:
it's more like a fraction. "Particle number per primary particle [(none)]".

"particle number" is according to ICRU 60 definition.

Reply:
What about "Particle multiplicity" or "Particle number / primary" ?

Reply:
"Particle number / primary" is a bit colloquial, but probably ok compromise.

Reply:
![image](https://user-images.githubusercontent.com/7374733/39548257-511afc32-4e5a-11e8-996e-a550401f64b2.png)
Finally